packages/domain-picker/src/domain-picker/use-your-domain-item.tsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { __ } from '@wordpress/i18n';
+import { ArrowButton } from '@automattic/onboarding';
+
+/**
+ * Internal dependencies
+ */
+
+interface Props {
+	onClick: () => void;
+}
+
+const UseYourDomainItem: React.FunctionComponent< Props > = ( { onClick } ) => {
+	return (
+		<label className="domain-picker__suggestion-item contains-link">
+			<div className="domain-picker__suggestion-item-name">
+				<span className="domain-picker__domain-wrapper with-margin with-bold-text">
+					{ __( 'Already own a domain?', __i18n_text_domain__ ) }
+				</span>
+				<div>
+					<span className="domain-picker__item-tip">
+						{ __( "You can use it as your site's address", __i18n_text_domain__ ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **30** times:
`translate( 'You can use it as your site's address.', { context: 'Upgrades: Register domain description'} )` *ES Score: 8*